corehq/apps/reports/standard/cases/case_list_explorer.py

```diff
@@ -91,7 +91,7 @@ def es_results(self):
     def _build_query(self, sort=True):
         query = super(CaseListExplorer, self)._build_query()
         query = self._populate_sort(query, sort)
-        query = self.apply_xpath_case_search_filter(self, query)
+        query = self.apply_xpath_case_search_filter(query)
```

Comment:
Bad miss from my end. Glad I could catch it.

Reply:
Ah, good one